js-api-examples/compile-streaming.html

```diff
@@ -11,17 +11,26 @@
 
 
     <script>
-      var importObject = { imports: { imported_func: arg => console.log(arg) } };
-
-      // fetch('simple.wasm')
+      const importObject = {
+        imports: {
+          imported_func: arg => {
```

Comment:
I believe all these changes of wrapping lambda-function results in curly braces need to be reverted: `x => x` returns `x` while `x => { x }` returns `undefined`.

Reply:
Ah, right you are!  Sorry for the confusion!